tools/testing/target_determination/heuristics/hipri.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Any
+
+from tools.testing.target_determination.heuristics.interface import (
+    HeuristicInterface,
+    TestPrioritizations,
+)
+
+from tools.testing.test_run import TestRun
+
+
+class HiPri(HeuristicInterface):
+    # Literally just a heuristic for test_public_bindings.  Pretty much anything
+    # that changes the public API can affect this test
+    hi_pri = [
+        "test_public_bindings",
```

Comment:
I guess this means that `test_public_bindings` will always runs.  Maybe it is more accurate to say that `test_public_bindings` runs when the public API changes, which means files under `torch` directory is updated (IIRC, that's where the public API is)

Reply:
Changed to look for changes in torch folder